tests/indexes/tests.py

```diff
@@ -226,11 +226,9 @@ def test_no_index_for_foreignkey(self):
                 new_field.set_attributes_from_name('new_foreign_key')
                 editor.add_field(ArticleTranslation, new_field)
                 field_created = True
-                self.assertEqual([str(statement) for statement in editor.deferred_sql], [
-                    'ALTER TABLE `indexes_articletranslation` '
-                    'ADD CONSTRAINT `indexes_articletrans_new_foreign_key_id_d27a9146_fk_indexes_a` '
-                    'FOREIGN KEY (`new_foreign_key_id`) REFERENCES `indexes_article` (`id`)'
-                ])
+                # There should be no deferred SQL as the FK constraint was
```

Comment:
It looks to me like this test should verify the SQL rather than verifying that there isn't any deferred SQL.